WooCommerce/Classes/ViewRelated/Bookings/Booking Details/BookingDetailsView.swift

```diff
@@ -0,0 +1,160 @@
+import SwiftUI
+import WooFoundation
+import Networking
+
+struct BookingDetailsView: View {
+    @Environment(\.safeAreaInsets) var safeAreaInsets: EdgeInsets
+
+    @ObservedObject private var viewModel: BookingDetailsViewModel
+
+    private enum Layout {
+        static let contentSidePadding: CGFloat = 16
+        static let headerContentVerticalPadding: CGFloat = 6
+        static let headerBadgesAdditionalTopPadding: CGFloat = 4
+    }
+
+    fileprivate enum TextFont {
+        static var bodyMedium: Font {
+            Font.body.weight(.medium)
+        }
+
+        static var bodyRegular: Font {
+            Font.body.weight(.regular)
+        }
+    }
+
+    init(_ viewModel: BookingDetailsViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        RefreshablePlainList(action: {
```

Comment:
Nit: this type seems outdated. Since we're targeting iOS 17 minimum now, please consider using `ScrollView` with `refreshable` for simplicity.